Esterni/ManualeUtente/components/c4-Istruzioni_per_utilizzo.tex

```diff
@@ -0,0 +1,50 @@
+\section{Istruzioni per l'utilizzo}
+\subsection{Autenticazione}
+All'avvio verrà mostrata una schermata che servirà ad autenticarsi, nella quale bisognerà inserire nome utente e password del proprio account \gloman{Zextras Drive}. In caso di successo dell'operazione si avvierà il software, altrimenti verrà visualizzato un messaggio di errore e chiesto di reinserire le credenziali.
```

Comment:
Cambierei
' All'avvio verrà mostrata una schermata che servirà ad autenticarsi'
in
'All'avvio verrà mostrata una schermata di autenticazione'

Reply:
Io lascerei così, dobbiamo pensare che l'utente non sappia fare nulla nella sua vita, quindi essere più espliciti possibile